client/question/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -63,22 +92,27 @@ tie.factory('PythonPrereqCheckService', [
      *    WRONG_LANGUAGE_ERRORS.
      */
     var detectAndGetWrongLanguageType = function(code) {
-      var codeLines = getNonStringLines(code);
+      var rawCodeArray = obscureStringCharacters(code).split('\n');
```

Comment:
Is it possible for a string to contain a (non-escaped) newline character? If so, would that mess up the line numbers?